src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Circle;
+import core.basesyntax.model.Color;
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.IsoscelesTrapezoid;
+import core.basesyntax.model.Rectangle;
+import core.basesyntax.model.RightTriangle;
+import core.basesyntax.model.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_COUNT_FIGURES = 5;
+    private static final int MAX_RANDOM_VALUE = 20;
+    private final Figure circle = new Circle(10, Color.WHITE.name());
```

Comment:
```suggestion
    private final Figure circle = new Circle(10, Color.WHITE.name());
```

you don't need variable with default figure, return this from method, but 10 and white move to the class constant

Reply:
As far as I right understanded from task, one part of figures must make random and it relate to circle (so, circle in this part should have random parameters), and other part only circle with constant parameters, is not right?

Reply:
or you wrote about make change these parameters into the class?